.goreleaser.develop.yaml

```diff
@@ -75,7 +75,6 @@ dockers:
       - --build-arg=COMMIT_SHA={{ .FullCommit }}
       - --build-arg=CL_MEDIAN_CMD=chainlink-feeds
       - --build-arg=CL_MERCURY_CMD=chainlink-mercury
-      - --build-arg=CL_SOLANA_CMD=chainlink-solana
```

Comment:
We only want this change on the `-ccip` image declarations (and let's also keep `-ccip-plugins` as is)